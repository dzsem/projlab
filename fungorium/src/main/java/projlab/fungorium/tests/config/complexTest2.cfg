#0
ADD TECTON
#1
ADD TECTON
#2
ADD THREADKILLINGTECTON
#3
ADD TECTON
REGISTER NEIGHBOUR 0 1
REGISTER NEIGHBOUR 1 2
REGISTER NEIGHBOUR 2 3
REGISTER NEIGHBOUR 4 0
SET TECTONKILLCHANCE 100
#4
ADD MUSHROOMBODY 0 1
#5
ADD MUSHROOMTHREAD 0 1
#6
ADD MUSHROOMTHREAD 0 2
REGISTER CONNECTION 5 6
#7
ADD MUSHROOMBODY 0 3
#8
ADD MUSHROOMTHREAD 0 3
#9
ADD MUSHROOMTHREAD 0 2
REGISTER CONNECTION 8 9
#10
ADD INSECT 0 1
#11
ADD INSECT 0 3
