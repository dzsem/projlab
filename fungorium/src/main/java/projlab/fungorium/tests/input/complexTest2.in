#TESZTELÃ‰S
INSECT 10 MOVE 2
INSECT 10 CUTTHREAD 6
#12
MUSHROOMBODY 4 DISTRIBUTESPORES
SET EFFECTGENERATION 12 BLOCK
INSECT 10 EATSPORE
INSECT 10 CUTTHREAD 9
MUSHROOMTHREAD 9 EATINSECT
SHOW 10
NEXTROUND
SHOW 9
INSECT 11 MOVE 2
SET TECTONKILLCHANCE 0
#13
MUSHROOMTHREAD 8 GROW 2
NEXTROUND
NEXTROUND
NEXTROUND
NEXTROUND
INSECT 11 MOVE 2
