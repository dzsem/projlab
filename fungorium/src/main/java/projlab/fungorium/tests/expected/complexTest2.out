# SAVE output.txt hatására
# az output.txt-nek tartalmaznia kell az alábbi sorokat:

ERROR THE INSECT CANT CUT THREADS
ERROR NO SUCH OBJECT
ERROR NO SUCH OBJECT
ERROR NO CONNECTION BETWEEN TECTONS
